src/msg/async/net_handler.cc

```diff
@@ -60,17 +63,20 @@ int NetHandler::create_socket(int domain, bool reuse_addr)
 int NetHandler::set_nonblock(int sd)
 {
   int flags;
+  int r = 0;
 
   /* Set the socket nonblocking.
    * Note that fcntl(2) for F_GETFL and F_SETFL can't be
    * interrupted by a signal. */
   if ((flags = fcntl(sd, F_GETFL)) < 0 ) {
-    lderr(cct) << __func__ << " fcntl(F_GETFL) failed: " << strerror(errno) << dendl;
-    return -errno;
+    r = -errno;
+    lderr(cct) << __func__ << " fcntl(F_GETFL) failed: " << cpp_strerror(r) << dendl;
+    return r;
```

Comment:
nit, could be more consistent: assign `errrno` to `r` (w/o negation), and then `return -r` (w/ negation)